src/documentation/gtag.ts

```diff
@@ -1,7 +1,7 @@
 'use strict';
 import {GTA3DocumentationProvider, docrequest, CommandDoc, GameDoc, ArgumentDoc} from './interface';
 import {GTA3ScriptController, Command} from '../controller';
-const cheerio = require('cheerio')
+const json = require('jsonify');
```

Comment:
Ah well, it was already in there in the `node_modules` dir as dependency for something so it caught my attention before realising TypeScript is just a more sensible JavaScript lol.